install/kubernetes/helm/istio-remote/templates/configmap.yaml

```diff
@@ -0,0 +1,114 @@
+    # convergence. Tweak this value according to your setup.
+    rdsRefreshDelay: {{ .Values.global.refreshInterval }}
+    #
+    defaultConfig:
+      # NOTE: If you change any values in this section, make sure to make
+      # the same changes in start up args in istio-ingress pods.
+      # See rdsRefreshDelay for explanation about this setting.
+      discoveryRefreshDelay: {{ .Values.global.refreshInterval }}
+      #
+      # TCP connection timeout between Envoy & the application, and between Envoys.
+      connectTimeout: 10s
+      #
+      ### ADVANCED SETTINGS #############
+      # Where should envoy's configuration be stored in the istio-proxy container
+      configPath: "/etc/istio/proxy"
+      binaryPath: "/usr/local/bin/envoy"
+      # The pseudo service name used for Envoy.
+      serviceCluster: istio-proxy
+      # These settings that determine how long an old Envoy
+      # process should be kept alive after an occasional reload.
+      drainDuration: 45s
+      parentShutdownDuration: 1m0s
+      #
+      # The mode used to redirect inbound connections to Envoy. This setting
+      # has no effect on outbound traffic: iptables REDIRECT is always used for
+      # outbound connections.
+      # If "REDIRECT", use iptables REDIRECT to NAT and redirect to Envoy.
+      # The "REDIRECT" mode loses source addresses during redirection.
+      # If "TPROXY", use iptables TPROXY to redirect to Envoy.
+      # The "TPROXY" mode preserves both the source and destination IP
+      # addresses and ports, so that they can be used for advanced filtering
+      # and manipulation.
+      # The "TPROXY" mode also configures the sidecar to run with the
+      # CAP_NET_ADMIN capability, which is required to use TPROXY.
+      #interceptionMode: REDIRECT
+      #
+      # Port where Envoy listens (on local host) for admin commands
+      # You can exec into the istio-proxy container in a pod and
+      # curl the admin port (curl http://localhost:15000/) to obtain
+      # diagnostic information from Envoy. See
+      # https://lyft.github.io/envoy/docs/operations/admin.html
+      # for more details
+      proxyAdminPort: 15000
+
+    {{- if .Values.global.remoteZipkinAddress }}
+      #
+      # Zipkin trace collector
+      zipkinAddress: zipkin.{{ .Release.Namespace }}:9411
+    {{- end }}
+
+    {{- if .Values.global.proxy.envoyStatsd.enabled }}
+      #
+      # Statsd metrics collector converts statsd metrics into Prometheus metrics.
+      statsdUdpAddress: {{ .Values.global.proxy.envoyStatsd.host }}:{{ .Values.global.proxy.envoyStatsd.port }}
+    {{- end }}
+
+    {{- if .Values.global.controlPlaneSecurityEnabled }}
+      #
+      # Mutual TLS authentication between sidecars and istio control plane.
+      controlPlaneAuthPolicy: MUTUAL_TLS
+      #
+      # Address where istio Pilot service is running
+      discoveryAddress: {{ .Values.global.remotePilotAddress }}:15005
```

Comment:
We need to fix this section for envoyv1 xDS v. envoyv2 xDS (grpc).  ports 15005 and 15007 are the envoy v1 (legacy) pilot endpoints.  The envoy v2 (grpc) endpoints are 15010 and 15011.

Reply:
yeah... I noticed that. Didn't want to deviate from the [main chart](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/configmap.yaml#L85-L99) though.

Reply:
Cool... good point.  Not sure if there's an issue already for this.

Reply:
There is an issue here https://github.com/istio/istio/pull/6452